lib/system/storage.toit

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+/**
+User-space side of the service API for key-value storage.
+*/
+
+import encoding.tison
+
+import system.api.storage show StorageService StorageServiceClient
+import system.services show ServiceResourceProxy
+
+_client_/StorageServiceClient? ::= (StorageServiceClient).open
+    --if_absent=: null
+
+class Bucket extends ServiceResourceProxy:
+  static SCHEME_RAM   ::= "ram"
+  static SCHEME_FLASH ::= "flash"
+
+  constructor.internal_ client/StorageServiceClient handle/int:
+    super client handle
+
```

Comment:
Doc comments to explain the uri format, what a Bucket is etc.

Reply:
Done.